src/Stryker.Core/Stryker.Core/Mutants/MutantOrchestrator.cs

```diff
@@ -60,21 +60,30 @@ public IEnumerable<Mutant> GetLatestMutantBatch()
         /// <returns>Mutated node</returns>
         public SyntaxNode Mutate(SyntaxNode currentNode)
         {
-            if (currentNode is StatementSyntax && currentNode.Kind() != SyntaxKind.Block)
+            if (GetExpressionSyntax(currentNode) is var expressionSyntax && expressionSyntax != null) {
```

Comment:
the is var pattern matching is not currently being used here. I think this whole if/elseif can be tidied up as the following:

```csharp
if (GetExpressionSyntax(currentNode) is var expressionSyntax && expressionSyntax != null)
{
    // The mutations should be placed using a ConditionalExpression
    ExpressionSyntax expressionAst = expressionSyntax;
    foreach (var mutant in FindMutants(expressionSyntax))
    {
        _mutants.Add(mutant);
        ExpressionSyntax mutatedNode = ApplyMutant(expressionSyntax, mutant);
        expressionAst = MutantPlacer.PlaceWithConditionalExpression(expressionAst, mutatedNode, mutant.Id);
    }
    return currentNode.ReplaceNode(expressionSyntax, expressionAst);
}
else if (currentNode is StatementSyntax ast && currentNode.Kind() != SyntaxKind.Block)
{
    // The mutations should be placed using an IfStatement
    foreach (var mutant in currentNode.ChildNodes().SelectMany(FindMutants))
    {
        _mutants.Add(mutant);
        StatementSyntax mutatedNode = ApplyMutant(currentNode as StatementSyntax, mutant);
        ast = MutantPlacer.PlaceWithIfStatement(ast, mutatedNode, mutant.Id);
    }
    return ast;
}
```